python-sdk/nuscenes/predict/input_representation/agents.py

```diff
@@ -0,0 +1,279 @@
+# nuScenes dev-kit.
+# Code written by Freddy Boulton, 2020.
```

Comment:
What is the standard license blurb we are using here? Especially important since we are releasing publically.

Reply:
I'm just going by what's in the other files. @holger-nutonomy is this header approved by the legal team?